spec/spec_helper.rb

```diff
@@ -1,5 +1,9 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
i think we have a pretty deep philosophical difference there. imo, i want to see that at minimum my code is staying covered between test runs. this is the professional consensus i've found, too.

i think the most noise is being added by the weird spec, which i can remove.

that being said, i have to rib you a little here, as you didn't even have tests -- so why do you mind the noise? ;)